fail2ban/server/failmanager.py

```diff
@@ -43,6 +44,33 @@ def __init__(self):
 		self.__maxRetry = 3
 		self.__maxTime = 600
 		self.__failTotal = 0
+		self.__subnetMask = None
+		
+	def setSubnetMask(self, value):
+		try:
+			self.__lock.acquire()
+			if "." in value: # 255.255.255.0 style mask
+				maskparts = value.split(".")
+				if len(maskparts) != 4:
+					raise ValueError("subnetmask must be valid")
+				for part in maskparts:
+					if int(part) < 0 or int(part) > 255:
+						raise ValueError("subnetmask must be valid")
+				self.__subnetMask = len(re.search("(?<=b)1+", bin(DNSUtils.addr2bin(value))).group())
```

Comment:
should or shouldn't we issue a warning whenever someone provides obscure netmask (e.g. by typo??) e.g. '254.255.255' which wouldn't resolve to binary leading with 1s and trailing with 0s?
imho we should and we should fail with a bang so smth like
```python
len(re.match("0b(1+)(0*)$", bin(DNSUtils.addr2bin(value)).group(1))
```
could be the pattern to look for while separating re.match and if None -- puking error?